questiontype.php

```diff
@@ -368,9 +377,26 @@ public function get_available_questions_from_category($categoryid, $subcategorie
         return $questionids;
     }
 
+    /**
+     * Gets all the questions from database with specific difficulty, tags and categoryid
+     * @param string    question difficulty
+     * @param string    question tags
+     * @param int       category id from $questiondata object
+     * @return array    ids of questions that fit the requirements
+     */
     public function get_available_questions_with_tags($difficulty, $topic, $categoryid) {
         global $DB;
-        $query = 'select tag_instance.itemid from {tag} tag join {tag_instance} tag_instance on tag.id = tag_instance.tagid where strcmp(upper(tag_instance.itemtype), \'QUESTION\') = 0 AND strcmp(upper(tag.name), upper("' . $difficulty . '")) = 0 intersect select tag_instance.itemid from {tag} tag join {tag_instance} tag_instance on tag.id = tag_instance.tagid where strcmp(upper(tag_instance.itemtype), \'QUESTION\') = 0 AND strcmp(upper(tag.name), upper("' . $topic . '")) = 0;';
+        $query = '
+            SELECT tag_instance.itemid
+            FROM {tag} tag
+                JOIN {tag_instance} tag_instance ON tag.id = tag_instance.tagid
+                WHERE strcmp(upper(tag_instance.itemtype), \'QUESTION\') = 0 AND strcmp(upper(tag.name), upper("' . $difficulty . '")) = 0
+            INTERSECT
+            SELECT tag_instance.itemid
+            FROM {tag} tag
+                JOIN {tag_instance} tag_instance ON tag.id = tag_instance.tagid
+                WHERE strcmp(upper(tag_instance.itemtype), \'QUESTION\') = 0 AND strcmp(upper(tag.name), upper("' . $topic . '")) = 0;
```

Comment:
`WHERE` clause should be at the same level of indentation as the `SELECT` clause